messaging/ses/src/sendEmail.ts

```diff
@@ -0,0 +1,82 @@
+import * as fetch from 'isomorphic-fetch';
+import * as FormData from 'form-data';
+import * as aws from 'aws-sdk';
+import { FunctionEvent } from 'graphcool-lib';
+
+const ses = new aws.SES({
+  region: process.env.AWS_REGION,
+  accessKeyId: 'AWS_ACCESS_KEY_ID',
+  secretAccessKey: 'AWS_SECRET_ACCESS_KEY',
+});
+
+
+interface EventData {
+  from: string
+  to: [string]
+  subject: string
+  text: string
+  html: string
+  recipientVariables: Object | null
+}
+
+
+
+
+export default async (event: FunctionEvent<EventData>) => {
+  console.log(event)
+  console.log('ENV:', process.env) 
+
+  if (!process.env['ACCESS_KEY_ID']) {
+    console.log('Please provide a valid AWS Access Key ID!');
+    return { error: 'Module not configured correctly.' }
+  }
+
+  if (!process.env['SECRET_ACCESS_KEY']) {
+    console.log('Please provide a valid AWS Secret Access Key!');
+    return { error: 'Module not configured correctly.' }
+  }
+
+  try {
+
+    const { to, from, subject, text, html } = event.data
+    const recipientVariables = event.data.recipientVariables || {}
```

Comment:
this is never used. is there an equivalent to `recipientVariables` with AWS SES?

Reply:
if not, remove this variable entirely (also from `sendEmail.graphql`)

Reply:
I don't believe so and I couldn't find anything on it in the docs. Granted, I can't ever find anything in AWS' docs. I'll try and do some more digging just to make sure